public/js/controllers/game.js

```diff
@@ -149,7 +149,7 @@ angular.module('mean.system')
       $scope.abandonGame = () => {
         game.leaveGame();
         gameTour.cancelTour();
-        $location.path('/');
+        window.location.href = '../';
```

Comment:
the removed one takes u home but maintains url of game but the new one fixes the issue and carries out a full redirect.